### ISSUE TITLE
TripletLoss

### ISSUE THREAD

**Issue by BabaSanfour** [timesler/facenet-pytorch#191]
In the mentioned paper of FaceNet they refer to the use of a Triplet loss to compare faces. I don't see this being reproduced in this repository. A simple CrossEntropyLoss is being used (in the fine-tuning notebook file). I am assuming the same was done for training? So it's not an accurate reproduction of the work done in the paper. 
Also, can someone mention where did they describe the ResInception architecture in the paper?

**Comment by gregworks** [comment]
I thought this was weird as well. David Sandberg discusses this in his opensource tensorflow version (also not trained with triplet loss): https://github.com/davidsandberg/facenet

**Comment by BabaSanfour** [comment]
Thanks! that was helpful.
